app/api/events.py

```diff
@@ -53,13 +53,13 @@
 
 def check_ticketing(user, modules):
     if not user.is_verified:
-        raise ForbiddenException({'source': '/data/attributes/is-ticketing-enabled'},
+        raise ForbiddenException({'source': '/data/attributes/is-verified'},
                                  "Please verify your Email")
     elif not modules.ticket_include:
-        raise ForbiddenException({'source': '/data/attributes/is-ticketing-enabled'},
+        raise ForbiddenException({'source': '/data/attributes/ticket-include'},
                                  "Ticket modules not included.")
     else:
-        raise ForbiddenException({'source': '/data/attributes/is-ticketing-enabled'},
+        raise ForbiddenException({'source': '/data/attributes/location-name'},
```

Comment:
This will run unconditionally

Reply:
@iamareebjamal What source should I specify in that case. Ticketing is not enabled according to location and user-verifed condition only AFAI could find.

Reply:
This is not about source. As you can see, the build has failed. You'll see the reason it failed is this line

Reply:
@iamareebjamal As the location is been checked in the later stages of the before_post function, I think we should just discard the else statement.

Reply:
If location is checked, then OK